src/core/Mixer.cpp

```diff
@@ -705,6 +704,7 @@ void Mixer::clearChannel(mix_ch_t index)
 	ch->m_volumeModel.setDisplayName( ch->m_name + ">" + tr( "Volume" ) );
 	ch->m_muteModel.setDisplayName( ch->m_name + ">" + tr( "Mute" ) );
 	ch->m_soloModel.setDisplayName( ch->m_name + ">" + tr( "Solo" ) );
+	ch->m_color = { };
```

Comment:
As Dom adviced me in Discord, it's probably more readable to write

```suggestion
	ch->m_color = std::nullopt;
```

This makes it clear to the reader that `ch->m_color` is an `std::optional`.